docs/content/standards/closed-loop-token/action-request.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: Action Request
+draft: true
+---
+
+Some actions are disabled by default. Token creator can enable them by creating a "TokenPolicy" and specifying conditions (Rules) for their execution.
+
+Token has 4 protected actions:
+
+| Function           | Action Name | Description                     |
+| ------------------ | ----------- | ------------------------------- |
+| `token::transfer`  | "transfer"  | Transfer a Token to a recipient |
+| `token::from_coin` | "from_coin" | Convert a Coin into a Token     |
+| `token::to_coin`   | "to_coin"   | Convert a Token into a Coin     |
+| `token::spend`     | "spend"     | Spend a Token                   |
+
+Protected actions are functions that issue an `ActionRequest` - a special hot-potato struct which must be "confirmed" for the transaction to succeed. ActionRequest is defined in the `sui::token` module and contains the following fields:
+
+- `name` - name of the performed action, standard ones are "transfer", "spend", "to_coin" and "from_coin"; and it is possible to create custom ones.
+- `amount` - the amount of the token that is being transferred, spent, converted, etc.
+- `sender` - the account that initiated the action
+- `recipient` - the account that receives the token in "transfer" action (can be used for custom actions)
+- `spent_balance` - the balance of a spent Token in the ["spend" action](./spending) (can be utilized in custom actions)
+
+These fields can be used by so-called "Rules" to determine whether the action should be allowed or not. Rules are custom modules which implement restriction logic. See "Rules" section for more details.
+
+An example of a function creating an `ActionRequest`:
+```move
+// module: sui::token
+public fun transfer<T>(
+    t: Token<T>, recipient: address, ctx: &mut TxContext
+): ActionRequest<T>;
+```
+
+## Creating a Custom Request
```

Comment:
Content about achieving a specific goal should be split out into the "Guides" section rather than "Standards", and then the two pages can be cross-linked.